internal/pkg/manifest/validate.go

```diff
@@ -1455,7 +1457,20 @@ func (q SQSQueueOrBool) validate() error {
 	if q.IsEmpty() {
 		return nil
 	}
-	return q.Advanced.validate()
+
+	if q.Enabled != nil {
+		return nil
+	} else {
+		if q.Advanced.Type != nil && strings.Compare(aws.StringValue(q.Advanced.Type), "fifo") == 0 {
```

Comment:
Why are we using `strings.Compare` instead of `==` 😛 ?

Reply:
I have added new func that checks weather `type` has correct value or not so we are good here. `==` will also works as we only allow small case `fifo` and `standard` as type values.